docker-database/docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: '3.8'
+services:
+  db:
+    container_name: database
+    build: DB_dockerDir/
+    # restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - '5432:5432'
+    volumes: 
+      - db:/var/lib/postgresql/data
+volumes:
+  db:
+    driver: local
```

```suggestion
    driver: local

```

and endline is required to the end of each file to be POSIX compliant (nitpick)

---

done